python/cudf/cudf/tests/test_timedelta.py

```diff
@@ -450,6 +453,7 @@ def test_timedelta_dataframe_ops(df, op):
         ),
     ],
 )
+@pytest.mark.filterwarnings("ignore::RuntimeWarning")
```

Comment:
I don't see any warnings from this test when running locally. What is this filter catching?

Reply:
I added it when I was seeing a warning, now I don't see it. My mistake.